src/v0/util/index.js

```diff
@@ -1305,6 +1306,55 @@ const generateExclusionList = (mappingConfig) =>
     Array.isArray(mapping.sourceKeys) ? [...mapping.sourceKeys] : [mapping.sourceKeys],
   );
 
+/**
+ * Generates an exclusion list using key paths based on the provided mapping configuration.
+ *
+ * @param {Array} mappingConfig - The traditional mapping configuration used build payloads.
+ * example :
+ * [
+ *     {
+ *       "destKey": "email",
+ *       "sourceKeys": "email",
+ *       "required": true,
+ *       "sourceFromGenericMap": true
+ *     },
+ *     {
+ *       "destKey": "key",
+ *       "sourceKeys": ["context.traits.key", "traits.key"],
+ *     }
+ *   ]
+ * @returns {Array} - The generated exclusion list, which can be fed in the "extractCustomFields" function.
+ */
+const generateExclusionListUsingKeyPaths = (mappingConfig) => {
```

Comment:
We already have exclusion related function in utils right ? Why do we need this again ?

Reply:
this is the exclusion key list, which will be required to be fed in `extractCustomFields`, otherwise we would need to make an array of constant type inside config.js.